wrangler.toml

```diff
@@ -42,3 +42,10 @@ enabled = true
 # [[services]]
 # binding = "MY_SERVICE"
 # service = "my-service"
+
+[[d1_databases]]
+binding = "DB"
+database_name = "byield"
+database_id = "db_id"
```

Comment:
what's the difference between name and ID?

Reply:
Here, the `database_id` will be the one that we will get from cloudflare D1.